src/main/webapp/app/exercises/quiz/manage/statistics/quiz-statistics-footer/quiz-statistics-footer.component.ts

```diff
@@ -138,11 +137,13 @@ export class QuizStatisticsFooterComponent implements OnInit, OnDestroy {
      * If the current page shows the first quiz question statistic then it will navigate to the quiz statistic
      */
     previousStatistic() {
+        const baseUrl = this.quizStatisticUtil.getBaseUrlForQuizExercise(this.quizExercise);
+
         if (this.isQuizStatistic) {
-            this.router.navigateByUrl(`/course-management/${getCourseId(this.quizExercise)}/quiz-exercises/${this.quizExercise.id}/quiz-point-statistic`);
+            this.router.navigateByUrl(baseUrl + `/quiz-point-statistic`);
```

Comment:
Why are you using + with string template? `${baseUrl}/quiz-point-statistic` or baseUrl + "quiz-point-statistic" please choose one :smile: